README.markdown

```diff
@@ -165,6 +165,12 @@ nnoremap <Leader>D :Thematic pencil_dark<CR>
 nnoremap <Leader>L :Thematic pencil_lite<CR>
 ```
 
+If you have installed
+[Yggdroot/LeaderF](https://github.com/Yggdroot/LeaderF), you can `:Leaderf
+thematic`.
+
+![integration with leaderf](screenshots/leaderf.png "integration with leaderf")
```

Comment:
Please drop the screen shot here. The README is already pretty cluttered and overwhelming. Also screenshots take up a ton of space and are frequently outdated. Adding support for LeaderF is fine, but the screenshot is a maintenance burden.

Reply:
I approve your opinion.

Reply:
i have removed all changes of README.